### ISSUE TITLE
bootp is disabled upon failover configuration, while this should not be

### ISSUE THREAD

**Issue by Rocco83** [voxpupuli/puppet-dhcp#226]
## Affected Puppet, Ruby, OS and module versions/distributions

- Module version: latest (3.3.0 at the time of bug reporting)

## How to reproduce (e.g Puppet code you use)

Configure DHCP with failover module

## What are you seeing

bootp is disabled by default, as by
https://github.com/voxpupuli/puppet-dhcp/pull/126

## What behaviour did you expect instead

The pull refer to https://linux.die.net/man/5/dhcpd.conf
But this chapter does not reflect anymore the current configuration, at least from Debian Stretch.

https://manpages.debian.org/testing/isc-dhcp-server/dhcpd.conf.5.en.html (ref: v4.3)
https://kb.isc.org/docs/isc-dhcp-41-manual-pages-dhcpdconf#configuring-failover
https://kb.isc.org/docs/isc-dhcp-44-manual-pages-dhcpdconf#configuring-failover

It should be sufficient to remove the bootp related line

**Comment by russellshackleford** [comment]
No, BOOTP should be disabled, at least by default, when failover is configured. While it is possible to find use cases when it is possibly safe to implement it, those cannot be assumed to be the default case. See https://tools.ietf.org/html/draft-ietf-dhc-failover-12#section-5.14.

However, I do think that disabling it as is done #126 is a bit heavy handed and an option should be offered, but the default should be disabled and the README should warn people not to enable it without understanding the consequences.
